django/contrib/admin/templates/admin/delete_selected_confirmation.html

```diff
@@ -20,14 +20,22 @@
 
 {% block content %}
 {% if perms_lacking %}
-    <p>{% blocktranslate %}Deleting the selected {{ objects_name }} would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktranslate %}</p>
+    <p>{% blocktranslate count counter=queryset.count %}
+      Deleting the selected {{ object_name }} would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:
+      {% plural %}
+      Deleting the selected {{ objects_name }} would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:
+    {% endblocktranslate %}</p>
     <ul>
     {% for obj in perms_lacking %}
         <li>{{ obj }}</li>
     {% endfor %}
     </ul>
 {% elif protected %}
-    <p>{% blocktranslate %}Deleting the selected {{ objects_name }} would require deleting the following protected related objects:{% endblocktranslate %}</p>
+    <p>{% blocktranslate count counter=queryset.count %}
```

Comment:
The changes of `blocktranslate` in this file might need to have the `trimmed` argument? Someone who knows more about the i18n machinery and Transifex can probably say with more certainty :)

Reply:
Yes, you are right. It is being rendered
```
    <p>
      Are you sure you want to delete the selected user? All of the following objects and their related items will be deleted:
      </p>
```
versus with `trimmed`:
```
    <p>Are you sure you want to delete the selected user? All of the following objects and their related items will be deleted:</p>
```
Let me make a fixup.

Reply:
Thanks!